gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,47 @@ def get_possible_database_path():
     return path
 
 
+def create_database_directory():
+    """Creates the standard GRASS GIS directory.
+
+    Creates database directory named grassdata in the standard location 
+    according to the platform.  
+    
+    Returns the new path as a string or None if nothing was found or created.
+    """
+    home = os.path.expanduser('~')     
+   
+    # Candidate for case independent "grassdata" for Linux and macOS
+    if sys.platform.startswith('linux') or sys.platform.startswith('darwin'):
+        candidate = os.path.join(home, "grassdata")
+    # Candidates for case independent "grassdata" in other dirs (Windows)
```

Comment:
The "case independent" does not make sense here. Simplify further, by merging the two comments together and saying something like "Determine the standard path according to a platform."

While you are at it, you can drop the "candidate" terminology from the variable name too. What candidate refers to here? It really makes sense only in the other function. Simple `path` will be probably more descriptive here. And going further, the `tmp` variable can be called `path` too because in a way it is no different from the `path` before, it replaces it for all intents and purposes.